projects/core/src/icon/icon.stories.ts

```diff
@@ -67,12 +68,102 @@ class AllIcons extends LitElement {
   }
 }
 
+@customElement('demo-icons-collection')
+class CollectionIcons extends LitElement {
+  @state() icons: string[] = [];
+  private collections: any = {};
+
+  private _collection: string;
+  get collection(): string {
+    return this._collection;
+  }
+
+  @property({ type: String })
+  set collection(val: string) {
+    this._collection = val;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    import('@cds/core/icon').then(module => {
+      module.loadChartIconSet();
+      module.loadCommerceIconSet();
+      module.loadCoreIconSet();
+      module.loadEssentialIconSet();
+      module.loadMediaIconSet();
+      module.loadMiniIconSet();
+      module.loadSocialIconSet();
+      module.loadTechnologyIconSet();
+      module.loadTextEditIconSet();
+      module.loadTravelIconSet();
+
+      this.collections = {
+        chart: module.chartCollectionIcons,
+        commerce: module.commerceCollectionIcons,
+        core: module.coreCollectionIcons,
+        essential: module.essentialCollectionIcons,
+        media: module.mediaCollectionIcons,
+        mini: module.miniCollectionIcons,
+        social: module.socialCollectionIcons,
+        technology: module.technologyCollectionIcons,
+        textEdit: module.textEditCollectionIcons,
+        travel: module.travelCollectionIcons,
+      };
+      this.icons = this.collections[this._collection].map((icon: IconShapeTuple) =>
+        ClarityIcons.getIconNameFromShape(icon)
+      );
+    });
+  }
+
+  render() {
+    return html`
+      <strong>${this._collection}</strong>
+      <div>
+        ${this.icons.map(
+          (icon, index) =>
+            html`
+              <div style="whitespace: no-wrap; display: inline-block; margin-right: .5rem;">
+                ${index + 1}.
+                <cds-icon
+                  size="lg"
+                  role="img"
+                  .shape=${icon}
```

Comment:
Just curious because the `AllIcons` component does this too. Why is this `.shape` instead of just `shape`.